sdk/python/requirements.txt

```diff
@@ -0,0 +1 @@
+kfp
```

Comment:
Do we want to have a minimum kfp version requirement?

Reply:
We could. I have not identified a technical minimum, but we could use the latest tested version which is 0.2.4

Reply:
sure, I was having 0.1.x version and some library path has changed since then.

Reply:
I set it to `>=0.2.2` which is the version tested earlier